src/modules/colorPicker/ColorPickerUI/Controls/ColorPickerControl.xaml

```diff
@@ -436,7 +436,9 @@
                                              AutomationProperties.Name="{x:Static p:Resources.Hex_value}"
                                              GotKeyboardFocus="HexCode_GotKeyboardFocus"
                                              TextChanged="HexCode_TextChanged"
-                                             TextWrapping="Wrap" />
+                                             TextWrapping="Wrap"
+                                             MaxLength="7"
+                                             CharacterCasing="Upper" />
```

Comment:
@crutkas 
I will change this to lower. Then we have the same layout of hex values everywhere. (I hope this is working when setting value from code too.)

Reply:
I have switched to lower case:
![image](https://user-images.githubusercontent.com/61519853/147226702-26e33a89-cd5c-4700-8de6-f504b22df96b.png)